src/mas/devops/templates/pipelinerun-install.yml.j2

```diff
@@ -713,3 +713,96 @@ spec:
     - name: shared-certificates
       secret:
         secretName: pipeline-certificates
+
```

Comment:
This section needs to be moved up into the params section, it's currently under workspaces.

Reply:
moved section